setup/CMakeLists.txt

```diff
@@ -8,6 +8,10 @@ project(dependencies VERSION 1 LANGUAGES C CXX)
 include(ExternalProject)
 include(CMakePrintHelpers)
 
+if(${CMAKE_VERSION} VERSION_EQUAL 3.24 OR ${CMAKE_VERSION} VERSION_EQUAL 3.25)
+  message(FATAL_ERROR "Protobuf build is broken in CMake version ${CMAKE_VERSION}")
```

Comment:
Should we specify the recommended version?
Since we have not tested with cmake version greater than 3.25, I am not sure if the bug is fixed in later versions

Reply:
Updated message text